nav-app/src/app/sidebar/sidebar.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { DataService } from '../data.service';
+import { ViewModel } from '../viewmodels.service';
+
+@Component({
+    selector: 'sidebar',
+    templateUrl: './sidebar.component.html',
+    styleUrls: ['./sidebar.component.scss']
```

Comment:
This technically violates the custom component prefix recommendation for Angular, but so do _most if not all_ of the Navigator custom components so it's probably not worth fixing in just this one case. We may want to do a short code cleanup sprint later to fix this.

https://angular.io/guide/styleguide#component-custom-prefix

Reply:
Yeah, that's a good point. @clemiller and I talked about this today, and I think we should wait until both our issues have been merged into the develop branch before renaming the sidebar component.